lib/ansible/modules/remote_management/oneview/oneview_datacenter.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/python
+# Copyright (c) 2016-2017 Hewlett Packard Enterprise Development LP
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: oneview_datacenter
+short_description: Manage OneView Data Center resources
+description:
+    - "Provides an interface to manage Data Center resources. Can add, update, and remove."
+version_added: "2.5"
+requirements:
+    - "python >= 2.7.9"
+    - "hpOneView >= 3.1.0"
+author:
+    - Alex Monteiro (@aalexmonteiro)
+    - Madhav Bharadwaj (@madhav-bharadwaj)
+    - Priyanka Sood (@soodpr)
+    - Ricardo Galeno (@ricardogpsf)
+options:
+    state:
+        description:
+            - Indicates the desired state for the Data Center resource.
+              C(present) will ensure data properties are compliant with OneView.
```

Comment:
Perhaps, 
`Will ensure the datacenter resource exists and its properties are compliant with HPE OneView, by creating or updating the resource.` 
Or something along those lines. As it is right now I'm not sure if it is clear that this resource can actually be created if it does not exist.